src/plugins/hiddenColumns/hiddenColumns.js

```diff
@@ -220,6 +255,17 @@ class HiddenColumns extends BasePlugin {
     return this.hiddenColumns.indexOf(columnIndex) > -1;
   }
 
+  /**
+   * Check whether all of the provided column indexes are within the bounds of the table.
+   *
+   * @param {Array} columns Array of column indexes.
+   */
+  isColumnDataValid(columns) {
+    const outOfBounds = columns.find(column => (column < 0 || column > this.hot.countCols() - 1));
```

Comment:
You may use also [every](https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/every), but decission about using it is up to you.